### ISSUE TITLE
Doc for why and how to avoid large cookies

### ISSUE THREAD

**Issue by robrap** [edx/edx-arch-experiments#165]
This document could potentially be short and sweet, but people should know why we need to avoid large cookies (or too many of cookies), and how they can be avoided.

It is possible that https://2u-internal.atlassian.net/wiki/spaces/AT/pages/16385315/Cookie+Header+Size+Limits could be used for this purpose, but I'm imagining a short-and-sweet doc that caters to helping other teams avoid future issues.

Acceptance Criteria:
- [x] Create new doc.
- [x] Share appropriately.
  - [x] Send announcements
  - [ ] Tech All-Hands (ticket blocked waiting on getting this scheduled)

**Comment by robrap** [comment]
Question: Do we want a separate ticket for an allow list on large cookies?
- We could include every last cookie, but maybe that is more on-call overhead than anyone would want. Using a threshold for what is considered a large cookie could make this simpler to manage.
- It would provide an early warning as soon as a new large cookie were introduced. This would hopefully allow us to provide feedback to someone while they are still working on something, versus the possibility of months or years of delay.
- This ticket for documentation and education feels more important, because it would be better for people to avoid the problem in the first place, but we may want both.

**Comment by robrap** [comment]
Note: The Feb 2 All Tech call might be a great place for a plug on sharing about cookies.

**Comment by robrap** [comment]
@rgraber: I did a revamp of the Confluence doc. You can either see the diffs: https://2u-internal.atlassian.net/wiki/pages/diffpagesbyversion.action?pageId=16385315&selectedPageVersions=10&selectedPageVersions=8, or review the final version of the page. Thanks.

**Comment by rgraber** [comment]
Made one small change to better emphasize what will happen if we go over 16k. Otherwise I'm happy with the doc as is.

**Comment by rgraber** [comment]
Blocking this on ensuring we get a slot at the next tech-all.
